events-api/src/modules/tickets/tickets.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from "express";
+import { TicketsService } from "./tickets.service";
+import { validateReqBody } from "#/shared/validators/request-body.validator";
+import { ticketReqBodyDtoSchema } from "./requests/ticket-req-body.dto";
+import { notFoundMiddleware } from "#/shared/middlewares/not-found.middleware";
+import { validateId } from "#/shared/validators/id-check.valildator";
+import { ticketIdDtoSchema } from "./requests/ticket-id.dto";
+
+export const TicketController = Router();
+
+TicketController.get(
+  "/:ticketId",
+  validateId(ticketIdDtoSchema),
+  async (req, res) => {
+    const ticketId = req.params["ticketId"] as string;
+
+    const ticket = await TicketsService.getOne(ticketId);
+
+    if (!ticket) {
+      return notFoundMiddleware(req, res);
```

Comment:
Wrong error handling, better to throw an error and handle it on the level of error middleware